base-api/src/main/java/com/yapp/crew/controller/AddressListController.java

```diff
@@ -0,0 +1,29 @@
+package com.yapp.crew.controller;
+
+import com.yapp.crew.dto.EnumListDto;
+import com.yapp.crew.dto.EnumListFailDto;
+import com.yapp.crew.dto.EnumListSuccessDto;
+import com.yapp.crew.service.EnumToList;
+import java.util.List;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AddressListController {
+
+  @GetMapping("/v1/address/city")
```

Comment:
```suggestion
  @GetMapping(path = "/v1/address/city")
```
이런식으로 앞에 접두사? 를 붙여주시면 더 보기가 좋을것 같네요